### DISCUSSION TITLE
account was flagged #2826012 and #2810581

### DISCUSSION THREAD

**Discussion by cyber-loafing** [community/community#129444] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My form has been submitted for over a month, yet my account is still inaccessible. I believe I have described in detail and sincerely in both forms. I really request you not to ban my account anymore, as I have very important personal assignments and a blog on that GitHub account. My issue has been elaborated in detail in the previous forms. Please unban it as soon as possible, thank you.

Additionally, after I sent the second form, I can no longer log into my account. Is there any specific reason for this?

**Comment by priyankarpal** [comment]
what happened to your account? @cyber-loafing

**Reply by cyber-loafing** [reply]
In May of this year, my account was compromised by someone else. When I discovered it, they had frantically forked many strange repositories. I have already included screenshots of the IP addresses they used in the form I submitted, which are completely inconsistent with my location. Then, I submitted an appeal in May, but there has been no response up to now. Previously, my account was only flagged, but now I can't even log in.

**Reply by cyber-loafing** [reply]
The name of that account is @Cyberloafingg

**Reply by priyankarpal** [reply]
> In May of this year, my account was compromised by someone else. When I discovered it, they had frantically forked many strange repositories. I have already included screenshots of the IP addresses they used in the form I submitted, which are completely inconsistent with my location. Then, I submitted an appeal in May, but there has been no response up to now. Previously, my account was only flagged, but now I can't even log in.

I suggest doing a tweet and tagging GitHub and their team. It could be helpful for you.

**Reply by priyankarpal** [reply]
> The name of that account is @Cyberloafingg

page not found 404

**Reply by cyber-loafing** [reply]
> > The name of that account is @Cyberloafingg
> 
> page not found 404
Yes, it is now banned, thank you for your suggestion.

**Comment by LiteBrite82** [comment]
Hi @cyber-loafing 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by Budalebah** [reply]
please help. me to ue my account , because of vercel my accunt flagged, help me plase
